src/components/ModalWindow/ModalWindow.module.scss

```diff
@@ -0,0 +1,63 @@
+@import 'styles/colors';
+@import 'styles/fontMixin';
+
+.modalList {
+    padding: 8px 16px 16px;
+    background: $grey;
+    border-radius: 0px 0px 5px 5px;
+    position: absolute;
+    left: 0;
+    top: 100%;
+    z-index: 999;
+    animation-name: active;
+    animation-duration: 0.4s;
```

Comment:
as i remember we have variable for `animation-duration`?

Reply:
was for transition, added for animation-duration